sdk/documenttranslation/Azure.AI.Translator.DocumentTranslation/tests/TranslationOperationLiveTests.cs

```diff
@@ -276,13 +282,23 @@ public async Task RightSourceWrongTarget()
             var input = new DocumentTranslationInput(source, target, "fr");
             DocumentTranslationOperation operation = await client.StartTranslationAsync(input);
 
-            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () => await operation.UpdateStatusAsync());
-
-            Assert.AreEqual("InvalidDocumentAccessLevel", ex.ErrorCode);
+            try
+            {
+                Console.WriteLine("Trying");
+                await operation.UpdateStatusAsync();
+                Console.WriteLine("Worked");
```

Comment:
@JoshLove-msft is there a way for me to output stuff when a test passes? So in theory this should throw an exception, but because the test sometimes passes I want to see what the service is returning

Reply:
TestContext.Out.WriteLine

Reply:
In order to see the logs, they should be there in the test output when you run in Live mode or have a debugger attached.

Reply:
Yeah, I need it in CI. These tests are flaky and haven't been able to reproduce the failure locally, so I am hoping that by adding some logging into CI I can get an understanding of what is happening

Reply:
Enabled by using ClientDiagnostics. Thanks Josh!